oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/resourceserver/web/BearerTokenAuthenticationFilter.java

```diff
@@ -143,8 +141,13 @@ protected void onUnsuccessfulAuthentication(HttpServletRequest request, HttpServ
 	 * @param bearerTokenResolver the {@code BearerTokenResolver} to use
 	 */
 	public void setBearerTokenResolver(BearerTokenResolver bearerTokenResolver) {
```

Comment:
add `final`

Reply:
Not sure what this comment applies to as it refers to line that wasn't changed in this PR.

Reply:
oh yes, sorry about that